backend/unit/unit_test.go

```diff
@@ -0,0 +1,182 @@
+package unit
+
+import (
+	"testing"
+	"time"
+
+	"github.com/short-d/app/mdtest"
+)
+
+func TestParseDuration(t *testing.T) {
+	t.Parallel()
+
+	t.Run("second", func(t *testing.T) {
+		testCases := []struct {
+			name             string
+			time             string
+			expectedHasError bool
+			expectedDuration time.Duration
+		}{
+			{
```

Comment:
We should also add cases for `empty string` and `multi digits` parsing ( 10s, 120s )

Reply:
We should also add a case for custom define `Duration` struct to make sure it fails because it's not the duration from the built-in `time` package.